recipe/meta.yaml

```diff
@@ -23,11 +22,13 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - setuptools_scm
   run:
     - python
-    - dask-core >=2023.2.0
-    - numpy >=1.18
-    - scipy >=0.19.1
+    - dask-core >=2024.4.1
+    # Tests fail without numpy upper bound
+    - numpy >=1.18,<2.0
```

Comment:
Can you add as a comment this tracking issue? 
https://github.com/dask/dask-image/issues/392